apps/frontend/src/hooks/aave/useAaveReservesData.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  ReserveDataHumanized,
+  UiPoolDataProvider,
+} from '@aave/contract-helpers';
+import { formatReserves, FormatReserveUSDResponse } from '@aave/math-utils';
+
+import { useCallback, useEffect, useState } from 'react';
+
+import dayjs from 'dayjs';
+
+import { getProvider } from '@sovryn/ethers-provider';
+
+import { BOB_CHAIN_ID } from '../../config/chains';
+
+import { config } from '../../constants/aave';
+import { ETH, WETH } from '../../constants/currencies';
+
+export type Reserve = ReserveDataHumanized & FormatReserveUSDResponse;
+export type ReserveData = { reserves: Reserve[]; loading: boolean };
+
+const uiPoolDataProvider = new UiPoolDataProvider({
+  provider: getProvider(BOB_CHAIN_ID),
+  uiPoolDataProviderAddress: config.UiPoolDataProviderV3Address,
+  chainId: Number(BOB_CHAIN_ID),
+});
+
+export const useAaveReservesData = (): ReserveData => {
+  const [reserves, setReserves] = useState<Reserve[]>([]);
+  const [loading, setLoading] = useState(false);
+
+  const fetchReservesData = useCallback(async () => {
+    const currentTimestamp = dayjs().unix();
+    const reservesData = await uiPoolDataProvider.getReservesHumanized({
+      lendingPoolAddressProvider: config.PoolAddressesProviderAddress,
+    });
+    const formattedReserves = formatReserves({
+      reserves: reservesData.reservesData,
+      currentTimestamp,
+      marketReferenceCurrencyDecimals:
+        reservesData.baseCurrencyData.marketReferenceCurrencyDecimals,
+      marketReferencePriceInUsd:
+        reservesData.baseCurrencyData.marketReferenceCurrencyPriceInUsd,
+    });
+
+    setReserves(
+      // rename weth to eth. We don't expose WETH through ui.
+      formattedReserves.map(r =>
+        r.symbol === WETH ? { ...r, symbol: ETH } : r,
+      ),
+    );
+  }, [setReserves]);
```

Comment:
is this conversion ok or just for our testnet case?

Reply:
Yes should be okay, we don't expose weth as save does so totally avoid the distinction in the frontend